docs/content/querying/lookups.md

```diff
@@ -27,537 +28,4 @@ and such data belongs in the raw denormalized data for use in Druid.
-Usage of these endpoints is quite advanced and not recommended for most users.
-The endpoints are as follows:
-
-## Get Lookups
-
-A `GET` to the node at `/druid/listen/v1/lookups` will return a json map of all the lookups currently active on the node.
-The return value will be a json map of the lookups to their extractor factories.
-
-```json
-
-{
-    "some_lookup_name": {
-        "type": "simple_json",
-        "uri": "http://some.host.com/codes.json"
-    }
-}
-
-```
-
-## Get Lookup
-
-A `GET` to the node at `/druid/listen/v1/lookups/some_lookup_name` will return the LookupExtractorFactory for the lookup identified by `some_lookup_name`.
-The return value will be the json representation of the factory.
-
-```json
-{
-    "type": "simple_json",
-    "uri": "http://some.host.com/codes.json"
-}
-```
-
-## Bulk Add or Update Lookups
-
-A `POST` to the node at `/druid/listen/v1/lookups` of a JSON map of lookup names to LookupExtractorFactory will cause the service to add or update its lookups.
-The return value will be a JSON map in the following format:
-
-```json
-{
-    "status": "accepted",
-    "failedUpdates": {}
-}
-
-```
-
-If a lookup cannot be started, or is left in an undefined state, the lookup in error will be returned in the `failedUpdates` field as per:
-
-```json
-{
-    "status": "accepted",
-    "failedUpdates": {
-        "country_code": {
-            "type": "simple_json",
-            "uri": "http://some.host.com/codes.json"
-        }
-    }
-}
-
-```
-
-The `failedUpdates` field of the return value should be checked if a user is wanting to assure that every update succeeded.
-
-## Add or Update Lookup
-
-A `POST` to the node at `/druid/listen/v1/lookups/some_lookup_name` will behave very similarly to a bulk update.
-
-If `some_lookup_name` is desired to have the LookupExtractorFactory definition of 
-
-```json
-{
-    "type": "simple_json",
-    "uri": "http://some.host.com/codes.json"
-}
-```
-
-Then a post to `/druid/listen/v1/lookups/some_lookup_name` will behave the same as a `POST` to `/druid/listen/v1/lookups` of
-
-```json
-
-{
-    "some_lookup_name": {
-        "type": "simple_json",
-        "uri": "http://some.host.com/codes.json"
-    }
-}
-
-```
-
-## Remove a Lookup
-A `DELETE` to `/druid/listen/v1/lookups/some_lookup_name` will remove that lookup from the node. Success will reflect the ID.
-
-# Configuration
-See the [coordinator configuration guilde](../configuration/coordinator.html) for coordinator configuration
-
-To configure a Broker / Router / Historical / Peon to announce itself as part of a lookup tier, use the `druid.zk.paths.lookupTier` property.
-
-|Property | Description | Default |
-|---------|-------------|---------|
-|`druid.lookup.tierName`| The tier for **lookups** for this node. This is independent of other tiers.|`__default`|
+For available lookups, please see our [extensions list](../development/extensions.html).
```

Comment:
This sentence isn't quite right… there are some lookups available without extensions. Maybe it should read, "for lookups defined cluster-wide rather than embedded in the query, please look at the namespaced lookup extension"